themes/bootstrap3/templates/_ui/components/confirm-menu.phtml

```diff
@@ -0,0 +1,46 @@
+<?php
+  if (!isset($this->label)) {
+    throw new \Exception('label undefined for confirm-menu component.');
+  }
+
+  $toggleLabel = $this->transEsc($this->label);
```

Comment:
I still think adding an Esc suffix to all these escaped variables might help reassure the casual reader that the variables are properly escaped in the template part below.